lib/rules/jsx-closing-bracket-location.js

```diff
@@ -198,6 +198,7 @@ module.exports = {
       }
       const openingLine = sourceCode.lines[opening.line - 1];
       const openingStartOfLine = {
+        // @ts-ignore
```

Comment:
why?

Reply:
I missed to clean up this after disabling  `--strictNullCheck`. Tsc cannot infer that `/^\s*/.exec(…)` is never null.

Reply:
Removed.